DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -125,10 +128,31 @@ struct NetworkProtectionStatusView: View {
         }
     }
 
+    @ViewBuilder
```

Comment:
TIL we don't need to return an empty view in the else case when using @ViewBuilder.